src/config/mongodb.ts

```
@@ -6,18 +6,57 @@ let client: MongoClient | null = null;
+        critical: false,
+      },
+      {
+        spec: { expiresAt: 1 },
+        options: { name: "idx_expiresAt_ttl", expireAfterSeconds: 0 },
+        critical: true, // TTL failure impacts security logic
+      },
+    ];
+
+    // Execute all index creations regardless of individual failures
+    const results = await Promise.allSettled(
+      indexConfigs.map((idx) => collection.createIndex(idx.spec, idx.options))
```

ESLint

[error] Insert `,`

(prettier/prettier)